base/applications/dxdiag/lang/es-ES.rc

```diff
@@ -9,9 +9,9 @@ CAPTION "Programa de diagnóstico de ReactX"
 FONT 8, "MS Shell Dlg"
 BEGIN
     CONTROL "Tab1", IDC_TAB_CONTROL, "SysTabControl32", WS_TABSTOP, 2, 2, 474, 250
-    PUSHBUTTON "Ay&uda", IDC_BUTTON_HELP, 2, 260, 50, 14, WS_CHILD | WS_VISIBLE | WS_TABSTOP
+    PUSHBUTTON "Ay&uda", IDC_BUTTON_HELP, 2, 260, 50, 14, WS_CHILD | WS_VISIBLE | WS_TABSTOP | WS_DISABLED
```

Comment:
But question: why the (e.g. english...) translation had those buttons with WS_DISABLED? Is it because their associated functionality is not yet implemented in dxdiag?

Reply:
Following https://github.com/reactos/reactos/blob/master/base/applications/dxdiag/dxdiag.c DxDiagWndProc, functionality for both IDC_BUTTON_SAVE_INFO and IDC_BUTTON_HELP is not yet implemented, but just returns true.

If is is preferable, i may try to implement it and make that PR draft for now.

Reply:
OK then for this PR it's better to have the WS_DISABLED. If you're interested in implementing the corresponding features, do it in a separate PR.